ScoutSuite/providers/gcp/resources/iam/bindings_separation_duties.py

```diff
@@ -0,0 +1,68 @@
+from ScoutSuite.providers.base.resources.base import Resources
+from ScoutSuite.providers.gcp.facade.base import GCPFacade
+
+
+class BindingsSeparationDuties(Resources):
+    def __init__(self, facade: GCPFacade, project_id: str):
+        super().__init__(facade)
+        self.project_id = project_id
+
+    async def fetch_all(self):
+        raw_bindings = await self.facade.cloudresourcemanager.get_member_bindings(self.project_id)
+        binding_id, binding = await self._parse_binding_separation(raw_bindings)
+        self[binding_id] = binding
+        x=1
+
+    async def _parse_binding_separation(self, raw_bindings):
```

Comment:
@SophieDorval this is missing the `name` field, so the partial shows up with a blank name.